docs/_posts/2018-05-31-Setup.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+---
+This page will take you through the steps you need to do to build and run the sample app locally.
+
+## Prerequisites
+The app uses [Web Components](https://www.webcomponents.org/introduction), [lit-html](https://github.com/Polymer/lit-html), which is a small library for writing HTML templates with JavaScript string literals, and [lit-element](https://github.com/Polymer/lit-element), a small Web Component base class built on top of it.
+
+This app depends on several [npm](https://www.npmjs.com/) packages, which you must be able to install. Make sure that you've already installed [node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) before moving on to the next step. If you already have `node` installed, make sure you're using the latest version -- we recommend using `v8.0.0` and above. Note that at the moment `v10.0.0` is not supported (we're working on fixing this!).
+
+## Creating a new app
+To create a new app that uses the `pwa-starter-kit` template:
+```
+git clone --depth 1 https://github.com/Polymer/pwa-starter-kit my-app
+cd my-app
+```
+
+This will generate the initial project structure inside the `my-app` folder, which roughly looks like this:
+```
+my-app
+├── images
+|   └── ...
+└── src
+|   └── ...
+└── test
+|   └── ...
+├── index.html
+├── README.md
+├── package.json
+├── polymer.json
+├── manifest.json
+├── service-worker.js
+├── sw-precache-config.js
+├── ... (misc project config files)
+```
+Checkout the [folder structure](https://github.com/Polymer/pwa-starter-kit/wiki/3.-Configuring-and-personalizing#folder-structure) page for details on what each file is used for.
+
+### Installing dependencies
+To install the project's dependencies, run
+```
+npm install
+```
+
+You're now ready to run and see your app!
+
+### Run the app in development mode
+To run the app locally, run
+```
+npm start
+```
+
+This will start a local server on port `8081`. Open [http://localhost:8081](http://localhost:8081) to view your app in the browser. Note that this server can continue running as you're making changes to your application, which you will see if you refresh the browser tab.
+
+If the port is already taken on your computer, or if you need to change the default hostname (because you're using a Docker container, for example), you can configure them using command line arguments:
+```
+npm start -- --hostname 0.0.0.0 --port 4444
+```
+
+### Run the tests
+Check out the [Application testing](https://github.com/Polymer/pwa-starter-kit/wiki/7.-Application-testing) page for more information about the tests. For a quick way to run the tests, run
```

Comment:
Need to update all the links to other pages to relative paths, e.g.
`https://github.com/Polymer/pwa-starter-kit/wiki/7.-Application-testing`
to 
`/Application-testing`